app/controllers/landable/api/access_tokens_controller.rb

```diff
@@ -42,6 +48,18 @@ def find_own_access_token(id = params[:id])
       def asset_token_params
         params.require(:access_token).permit(:username, :password)
       end
+
+      def determine_permissions(user_groups) # input: ["HipChat Users", "PagerDuty Users", "popops"]
```

Comment:
Assignment Branch Condition size for determine_permissions is too high. [17.29/15]<br>Line is too long. [100/80]